tests/unittests/suchvim/user-plugin.vader

```diff
@@ -0,0 +1,44 @@
+Before (setUp):
+  source ~/.SUCH-Vim/suchvim/user-plugins.vim
+
+
+Execute (Given user plugin When add plugin Then plugin is added):
+  let plugin_to_add = "ValidGithubUserName/ValidPluginName"
+
+  let actual = SUCHVim_addPlugin(plugin_to_add)
+
+  let expected = "Plug '".plugin_to_add."'"
+  Assert expected == actual[0]
+
+
+Execute (Given users plugins When add plugin Then plugins is added):
+  let plugin1_to_add = "ValidGithubUserName/ValidPluginName"
+  let plugin_to_add2 = "ValidGithubUserName2/ValidPluginName2"
+
+  call SUCHVim_addPlugin(plugin_to_add)
+  let actual = SUCHVim_addPlugin(plugin_to_add2)
+
+  let expected = "Plug '".plugin_to_add."'"
+  let expected2 = "Plug '".plugin_to_add2."'"
+  Assert expected == actual[0]
+  Assert expected2 == actual[1]
+
+
+Execute (Given user plugin When add plugins Then plugin is added):
+  let plugin_to_add = ["ValidGithubUserName/ValidPluginName"]
+
+  let actual = SUCHVim_addPlugins(plugin_to_add)
+
+  let expected = "Plug '".plugin_to_add[0]."'"
+  Assert expected == actual[0]
+
+
+Execute (Given user plugins When add plugins Then plugins is added):
```

Comment:
Same comment here. This test or the previous could probably removed as they test the same thing.